src/main/java/org/opensrp/repository/postgres/LocationRepositoryImpl.java

```diff
@@ -731,6 +794,10 @@ private StructureMetadata createStructureMetadata(PhysicalLocation entity, Long
 			}
 		}
 		structureMetadata.setServerVersion(entity.getServerVersion());
+
+		if(id != null){
+			structureMetadata.setDateEdited(new Date());
+		}
```

Comment:
Do we need to add the complementary branch of this if condition for created at?

Reply:
https://github.com/OpenSRP/opensrp-server-core/blob/acbdf68075023cfb16a524f1ace88a6218c69b42/src/main/java/org/opensrp/repository/postgres/LocationRepositoryImpl.java#L192

Reply:
I think metadata never gets populated with date created in the _createMetadata_ method?